Source/msg.cpp

```diff
@@ -1434,7 +1434,8 @@ DWORD OnKnockback(const TCmd *pCmd, int pnum)
 	const auto &message = *reinterpret_cast<const TCmdParam1 *>(pCmd);
 
 	if (gbBufferMsgs != 1 && Players[pnum].isOnActiveLevel() && message.wParam1 < MaxMonsters) {
-		M_GetKnockback(message.wParam1);
+		Monster &monster = Monsters[message.wParam1];
+		M_GetKnockback(monster);

```

Comment:
If it's not used for anything else might as well just inline it.
```suggestion
		M_GetKnockback(Monsters[message.wParam1]);
```

Reply:
Done, thanks!